docs/editor_manual/finding_your_way_around/the_explorer_page.md

```diff
@@ -4,31 +4,22 @@ The Explorer page allows you to view a page's children and perform actions on th
 
 ![](../../_static/images/screen05_explorer_page.png)
 
--   The name of the section you are looking at is displayed below the breadcrumb (the row of page names beginning with the home icon). Each section is also itself a page (in this case the homepage). Clicking the title of the section takes you to the Edit screen for the section page.
+-   The name of the section you are looking at is displayed at the top. Each section is also itself a page (in this case the homepage). Clicking the title of the section takes you to the Edit screen for the section page.
 -   As the heading suggests, below are the child pages of the section. Clicking the titles of each child page will take you to its Edit screen.
-
-![](../../_static/images/screen06_explorer_page_arrow.png)
```

Comment:
Removed because this didn’t feel very valuable of a visual without the context of the page around it – which is shown on other screenshots anyway.